libs/helm/sidebar/src/lib/hlm-sidebar.ts

```diff
@@ -0,0 +1,115 @@
+import { ChangeDetectionStrategy, Component, computed, effect, inject, input } from '@angular/core';
+import { BrnSheetContent } from '@spartan-ng/brain/sheet';
+import { HlmSheet, HlmSheetContent } from '@spartan-ng/helm/sheet';
+import { hlm } from '@spartan-ng/helm/utils';
+
+import { HlmSidebarService, SidebarVariant } from './hlm-sidebar.service';
+
+import { NgTemplateOutlet } from '@angular/common';
+import type { ClassValue } from 'clsx';
+import { injectHlmSidebarConfig } from './hlm-sidebar.token';
+
+@Component({
+	selector: 'hlm-sidebar',
+
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	imports: [HlmSheet, HlmSheetContent, NgTemplateOutlet, BrnSheetContent],
+
+	template: `
+		<ng-template #contentContainer>
+			<ng-content></ng-content>
+		</ng-template>
+
+		@if (collapsible() === 'none') {
+			<div data-slot="sidebar" [class]="_nonCollapsibleComputedClass()">
+				<ng-container *ngTemplateOutlet="contentContainer"></ng-container>
+			</div>
+		} @else if (_sidebarService.isMobile()) {
+			<hlm-sheet
+				[side]="'left'"
```

Comment:
Use `side` input also for the sheet position

```suggestion
				[side]="'side()'"
```